ballerina/target/build

```diff
@@ -0,0 +1,8 @@
+{
+  "last_build_time": 1728879973748,
```

Comment:
Do we need this target directory, i think we need to gitignore this?

Reply:
fixed in `4e43da2`